src/main/kotlin/com/jetbrains/snakecharm/lang/parser/SmkTokenTypes.kt

```diff
@@ -42,27 +22,27 @@ object SmkTokenTypes {
     val WORKFLOW_ONERROR_KEYWORD = PyElementType("WORKFLOW_ONERROR_KEYWORD")
     val WORKFLOW_ONSTART_KEYWORD = PyElementType("WORKFLOW_ONSTART_KEYWORD")
     val WORKFLOW_TOPLEVEL_PYTHON_BLOCK_PARAMETER_KEYWORDS = TokenSet.create(
-            WORKFLOW_ONSUCCESS_KEYWORD, WORKFLOW_ONERROR_KEYWORD, WORKFLOW_ONSTART_KEYWORD
+        WORKFLOW_ONSUCCESS_KEYWORD, WORKFLOW_ONERROR_KEYWORD, WORKFLOW_ONSTART_KEYWORD
     )
 
     val WORKFLOW_TOPLEVEL_DECORATORS_WO_RULE_LIKE = TokenSet.orSet(
-            WORKFLOW_TOPLEVEL_PARAMLISTS_DECORATOR_KEYWORDS,
```

Comment:
Previous imp includes all top level sections (e.g `configfile:`) here. Toplevel sections token types were replaced with `WORKFLOW_TOPLEVEL_DECORATOR_KEYWORD`, but new impl doesn't include `WORKFLOW_TOPLEVEL_DECORATOR_KEYWORD`. E.g. that is why you have to hardcode `WORKFLOW_TOPLEVEL_DECORATOR_KEYWORD` where not need, e.g. in `SmkWordsScanner`. Please restore previous behaviour + review all usages of`WORKFLOW_TOPLEVEL_DECORATOR_KEYWORD`  and cleanup where you don't need it actually.

Reply:
https://github.com/JetBrains-Research/snakecharm/pull/405/commits/38d3b41b85c33d3f87cce080fa09d5b7a22db553